Localization/location.py

```diff
@@ -355,6 +355,17 @@ def signal_handler(signal, frame):
             calib.write_pos(out_dir,name,np.matrix(np.array(np.array(r_real)[0]).reshape((1,3))))
 
 #--------------------------- 4.1 Visual localization ----------------------#
```

Comment:
Why are these lines moved? If this is on purpose, please also move the comments.